<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NuevoAutor</class>
 <widget class="QWidget" name="NuevoAutor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Introducir nuevo autor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nombre</string>
       </property>
       <property name="buddy">
        <cstring>txtNombre</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtNombre"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Apellidos</string>
       </property>
       <property name="buddy">
        <cstring>txtApellidos</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtApellidos"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btCancelar">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btOK">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
